src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -30,36 +31,33 @@ import useHasActionPermissions from '../../hooks/permissions/useHasActionPermiss
 import useHasNoDecisionMethods from '../../hooks/permissions/useHasNoDecisionMethods.ts';
 import useActionFormProps from '../../hooks/useActionFormProps.ts';
 import useSidebarActionForm from '../../hooks/useSidebarActionForm.ts';
+import { type ActionFormBaseProps } from '../../types.ts';
```

Comment:
I think the above 4 imports would be better off being absolute imports. Generally I think it's good to have relative imports only for files that are in the same folder, or, at a stretch the parent folder. Any more than that and I start to get itchy 🤣 Happy to hear opinions if you disagree though!

(This also kinda matches what is written [here](https://github.com/JoinColony/ColonyFrontEndLivingStandard/blob/f09a6d489c6b496a067b34838dece3107da1a15e/repo/index.md?plain=1#L43))